project/en-US/jts/extras/XAConnectionRecovery.java

```diff
@@ -1,13 +1,6 @@
-
 /*
- * Copyright (C) 2000, 2001,
- *
- * Hewlett-Packard,
- * Arjuna Labs,
- * Newcastle upon Tyne,
- * Tyne and Wear,
- * UK.
- *
+ * Copyright The Narayana Authors
```

Comment:
I'm not convinced about that. These files in our docs are examples and I'd use the same argument for not including the copyright that we [used on the quickstart repo to justify not including them](https://github.com/jbosstm/quickstart/pull/438#pullrequestreview-1464911951).

The copyright change on this file is only because the script that I used for the main repo detected it and missed it when sanity checking the changes, I think it should be removed, we could ask for a third opinion?

Reply:
Yes, sure. BTW If you want to remove the copyright and license statement from this file I am ok with this PR. I agree with "the same argument for not including the copyright that we used on the quickstart repo to justify not including them".